macosx/Torrent.mm

```diff
@@ -863,8 +863,13 @@ - (CGFloat)availableDesired
 
 - (BOOL)isActive
 {
-    return self.fStat->activity != TR_STATUS_STOPPED && self.fStat->activity != TR_STATUS_DOWNLOAD_WAIT &&
-        self.fStat->activity != TR_STATUS_SEED_WAIT;
+    return self.fStat->peersSendingToUs > 0 || self.fStat->peersGettingFromUs > 0 || self.fStat->webseedsSendingToUs > 0 ||
+        self.fStat->activity == TR_STATUS_CHECK;
+}
```

Comment:
Another way to have avoiding the many file changes would have been to call this `isTransmitting` and leaving `isActive` as it was defined previously. That would have minimised the changes and make it more reviewable and safe.

Reply:
See comment for other review point :)